include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,553 @@
+
+    /* PrimitivesGeneratedQueryFeaturesEXT *//* VK_EXT_primitives_generated_query */
+    //VkBool32           primitivesGeneratedQuery;
+    //VkBool32           primitivesGeneratedQueryWithRasterizerDiscard;
+    //VkBool32           primitivesGeneratedQueryWithNonZeroStreams;
+
+    /* PrimitiveTopologyListRestartFeaturesEXT *//* VK_EXT_primitive_topology_list_restart */
+    //VkBool32           primitiveTopologyListRestart;
+    //VkBool32           primitiveTopologyPatchListRestart;
+
+    /* PrivateDataFeatures *//* VK_EXT_private_data *//* MOVED TO Vulkan 1.3 Core */
+
+    /* ProvokingVertexFeaturesEXT *//* VK_EXT_provoking_vertex */
+    //VkBool32           provokingVertexLast;
+    //VkBool32           transformFeedbackPreservesProvokingVertex;
+
+    /* Robustness2FeaturesEXT *//* VK_EXT_robustness2 */
+    //VkBool32           robustBufferAccess2;
+    //VkBool32           robustImageAccess2;
+    //VkBool32           nullDescriptor;
+    
+    /* ScalarBlockLayoutFeaturesEXT *//* VK_EXT_scalar_block_layout *//* MOVED TO Vulkan 1.2 Core */
+    
+    /* ShaderAtomicFloatFeaturesEXT *//* VK_EXT_shader_atomic_float */
+    //VkBool32           shaderBufferFloat32Atomics;
+    //VkBool32           shaderBufferFloat32AtomicAdd;
+    //VkBool32           shaderBufferFloat64Atomics;
+    //VkBool32           shaderBufferFloat64AtomicAdd;
+    //VkBool32           shaderSharedFloat32Atomics;
+    //VkBool32           shaderSharedFloat32AtomicAdd;
+    //VkBool32           shaderSharedFloat64Atomics;
+    //VkBool32           shaderSharedFloat64AtomicAdd;
+    //VkBool32           shaderImageFloat32Atomics;
+    //VkBool32           shaderImageFloat32AtomicAdd;
+    //VkBool32           sparseImageFloat32Atomics;
+    //VkBool32           sparseImageFloat32AtomicAdd;
+
+    /* ShaderAtomicFloat2FeaturesEXT *//* VK_EXT_shader_atomic_float2 */
+    //VkBool32           shaderBufferFloat16Atomics;
+    //VkBool32           shaderBufferFloat16AtomicAdd;
+    //VkBool32           shaderBufferFloat16AtomicMinMax;
+    //VkBool32           shaderBufferFloat32AtomicMinMax;
+    //VkBool32           shaderBufferFloat64AtomicMinMax;
+    //VkBool32           shaderSharedFloat16Atomics;
+    //VkBool32           shaderSharedFloat16AtomicAdd;
+    //VkBool32           shaderSharedFloat16AtomicMinMax;
+    //VkBool32           shaderSharedFloat32AtomicMinMax;
+    //VkBool32           shaderSharedFloat64AtomicMinMax;
+    //VkBool32           shaderImageFloat32AtomicMinMax;
+    //VkBool32           sparseImageFloat32AtomicMinMax;
+    
+    /* DemoteToHelperInvocationFeaturesEXT *//* VK_EXT_shader_demote_to_helper_invocation *//* MOVED TO Vulkan 1.3 Core */
+
+    /* ShaderImageAtomicInt64FeaturesEXT *//* VK_EXT_shader_image_atomic_int64 */
+    //VkBool32           shaderImageInt64Atomics;
+    //VkBool32           sparseImageInt64Atomics;
```

Comment:
expose, in GL/GLES they're always `false` as no extension adds a 64-bit per channel format for images

Reply:
unlike other "shader-only" extensions lets not enable by default because affects format promotion and validation